sqlalchemy_trino/datatype.py

```diff
@@ -156,3 +157,6 @@ def parse_sqltype(type_str: str) -> TypeEngine:
         type_kwargs = dict(timezone=type_str.endswith("with time zone"))
         return type_class(**type_kwargs)  # TODO: handle time/timestamp(p) precision
     return type_class(*type_args)
+
+if __name__ == "__main__":
+    parse_sqltype("row(min timestamp(6) with time zone, max timestamp(6) with time zone, null_count bigint)")
```

Comment:
Could you move this one into test-case bellow.

Reply:
fixed!